openfe/tests/protocols/test_openmm_equil_rfe_protocols.py

```diff
@@ -1407,7 +1407,7 @@ def test_get_estimate(self, protocolresult):
         est = protocolresult.get_estimate()
 
         assert est
-        assert est.m == pytest.approx(16.85, abs=0.3)
+        assert est.m == pytest.approx(15.71, abs=0.3)
```

Comment:
@hannahbaumann could you double check what value you get here when re-running the generator for this? I've always gotten ~ 16.8 for this sytem, it's really odd that things changed by a whole kcal/mol 🤯

Reply:
Just to add here - I ran this multiple times and consistently got ~ 15.7

Reply:
From today's call - we didn't guard against elf10 in partial charge generation...

Reply:
This definitely was down to oechem vs ambertools...